tests/foreman/ui/test_discoveryrule.py

```diff
@@ -711,13 +816,18 @@ def test_positive_view_existing_rule_with_non_admin_user(self):
 
         @Assert: Rule should be visible to non-admin user.
 
-        @caseautomation: notautomated
-
         @CaseLevel: Integration
         """
+        with Session(
+            self.browser, self.username, self.user_password
+        ) as session:
+            session.nav.go_to_discovery_rules()
+            locators['discoveryrules.disabled_name']
```

Comment:
Nop.. its not required at all.. somehow I missed to remove.. thank you for pointing that.